app/javascript/ColumnsEditorComponent/TwoColumns.test.jsx

```diff
@@ -0,0 +1,62 @@
+// XXX: This "mocks" include is required to monkey-patch some browser APIs not
+//      available in Node.
+import "./mocks";
+import createColumnsComponent from ".";
+
+describe("TwoColumns", () => {
+  const TwoColumns = createColumnsComponent({
+    label: "Two Columns",
+    columns: [
+      { name: "left", span: 8 },
+      { name: "right", span: 4 },
+    ],
+  });
+
+  const example = `
+Here is some preamble text
+
+{::columns span="8,4"}
```

Comment:
[question] It doesn't look like `span="8,4"` is supported? Or at least the netlify preview doesn't work with it. I think it's a little confusing to have a test documenting something that doesn't work elsewhere.